jenkins/jdbc-stage-maven-release.jenkinsfile

```diff
@@ -0,0 +1,80 @@
+lib = library(identifier: 'jenkins@main', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build.git',
+]))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2'
+            args '-e JAVA_HOME=/opt/java/openjdk-11'
+            alwaysPull true
+        }
+    }
+    triggers {
+        GenericTrigger(
+            genericVariables: [
+                [key: 'ref', value: '$.ref'],
+                [key: 'VERSION', value: '$.ref', regexpFilter: 'refs/tags/v' ],
+            ],
+            tokenCredentialId: 'jenkins-sql-generic-webhook-token',
+            causeString: 'A tag was cut on opensearch-project/sql repository causing this workflow to run',
+            printContributedVariables: false,
+            printPostContent: false,
+            regexpFilterText: '$ref',
+            regexpFilterExpression: '^refs/tags/.*'
```

Comment:
Due to sql has so many components in one repo.
We will have to use some pattern on tag to decide on which one to be triggered.
We have this practice in maps repo to follow here:

`sql-cli-release-<VERSION>`
`sql-jdbc-release-<VERSION>`
`sql-odbc-release-<VERSION>`

https://github.com/peterzhuamazon/maps/blob/a40788a611e1d0f573d715e0b1ce59b7fb33e4d0/jenkins/maps-server-release.jenkinsfile#L44-L45

Reply:
Fixed in e8d9a435.
Should I update `regexpFilter` on line 19 too?
https://github.com/Bit-Quill/opensearch-project-sql/blob/6ed5ecd2cfd7c619641af910ea54a5d010835967/jenkins/jdbc-stage-maven-release.jenkinsfile#L19